app/scripts/vendor/EARLY_VOTING_AddressPoints.geojson

```diff
@@ -84,6 +86,7 @@
         "Full_Addr": "51 Inman St",
         "Entry": "Primary",
         "TYPE": "BUILDING",
+        "LOCATION": "Department of Human Service Programs",
```

Comment:
Thanks for adding the location names, this is really helpful!
I found this page from Cambridge: https://www.cambridgema.gov/election/earlyvoting/earlyvotinginperson,
can you revise the names to match this?